app.rb

```diff
@@ -1,8 +1,59 @@
 require 'sinatra/base'
+require './lib/player'
+require './lib/game'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+  enable :sessions
+  set :session_secret, "there be dragons"
+
+  get '/' do
+    erb :index
+  end
+
+  post '/Home' do
+    session[:player1] = params[:player1]
+    session[:player2] = params[:player2]
+    redirect '/Player1'
+  end
+
+  get '/Player1' do
+    @player1 = Player.new(session[:player1])
+    @player2 = Player.new(session[:player2])
+    erb :play1
+  end
+
+  post '/player1_choice' do
+    session[:Rock] = params[:Rock]
+    session[:Paper] = params[:Paper]
+    session[:Scissors] = params[:Scissors]
+    redirect '/Player2'
   end
 
+  get '/Player2' do
+    @player2 = Player.new(session[:player2])
```

Comment:
this has already been defined in /player1_choice , is it meant to be redefined?

Reply:
good spot, it needs to be taken out of /player1_choice. It won't actually be or do anything in the first instance so it doesn't break anything